app/styles/components/_spinner.scss

```diff
@@ -0,0 +1,54 @@
+$duration: 0.5s;
+$spinner-color: $wds-color-link;
+$z-side-nav-overlay: $z-8;
+
+.wds-spinner {
+	@keyframes rotator {
+		from {
+			transform: rotate(0);
+		}
+		to {
+			transform: rotate(360deg);
+		}
+	}
+
+	@keyframes dash {
+		to {
+			stroke-dashoffset: 0;
+		}
+	}
+
+	animation: rotator $duration linear infinite;
+	left: 50%;
+	margin: -33px 0 0 -33px;
```

Comment:
I don't see size defined for this spinner. Are hardcoded values ok for every size? Maybe we should use `translate` to move it half left and top? Or maybe you'd use `flexbox`?